File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/04_compito_java_20150623/Orale.py
from Appello import Appello


class Orale(Appello):
    def __init__(self, tipo, codice, docente, corso, giorno, mese, anno, ora, max_studenti):
        super().__init__(tipo, codice, docente, corso, giorno, mese, anno, ora)
        self._max_studenti = max_studenti

    def __str__(self):
        return "Orale\t" + super().__str__() + "\t-\t-\t" + str(self._max_studenti)

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/04_compito_java_2015_06_23/Orale.java
public class Orale extends Appello {
    private int studenti;

    public Orale(int codice, String docente, String corso, int giorno, int mese, int anno, int ora, int studenti) {
        super(codice, docente, corso, giorno, mese, anno, ora);
        this.studenti = studenti;
    }

    public String toString() {
        return "orale" + "\t" + codice + "\t" + docente + "\t" + corso + "\t" + giorno + "\t" + mese + "\t" + anno + "\t" + ora + "\t-\t-\t" + studenti;
    }
}